src/lib/constants.ts

```diff
@@ -19,7 +19,9 @@ export const ICHI_ONEFOX_VAULT_API = 'https://api.ichi.org/v1/farms/20003'
 export const FEATURE_FLAGS = {
   airdrop: false,
   ethFoxStakingV1: true,
-  ethFoxStakingV2: true
+  ethFoxStakingV2: true,
+  // this must be set to true for testing
```

Comment:
No need to leave this comment in the code - if you'd like, you can add it to the PR description when describing how to test the changes.

Reply:
fixed